### ISSUE TITLE
Scrollbar does not work when a screen reader software is enabled.

### ISSUE THREAD

**Issue by Zerhar** [amcharts/amcharts5#113]
Scrollbar does not work when a screen reader software is enabled. Enable the "Narrator" app on Windows (Ctrl+windows button+Enter) and try to move the startGrip or the endGrip on the scrollbar with the arrow keys. It doesn't work and there is no way to configure the scrollbar buttons to be triggered by other keys.

**Comment by martynasma** [comment]
The issue with Windows Narrator is that it switches automatically to "scan mode" when in browser window.

It means that it basically hijacks all keyboard events (any JS keyboard event handlers do not work).

For other screen readers it's possible to disable scan mode on a chart by setting `role = "application"`, but Narrator seems to ignore that.

So, to my knowledge, the only solution is for user to disable scan mode (Narrator key + Space). Then everything starts working as you would expect.

P.S. narrator key is Caps Lock and/or Insert by default. Changeable in Windows settings.

If someone can suggest a way to "get trough" Narrator's keyboard event lockdown without requiring user to exit scan mode, please please please share the knowledge. The beer is on me :)

**Comment by Malvoz** [comment]
> For other screen readers it's possible to disable scan mode on a chart by setting role = "application", but Narrator seems to ignore that.

I have [reported the issue to Microsoft](https://answers.microsoft.com/en-us/windows/forum/windows_11-wintop_apps-win_subtopwinapps/bug-narrator-ignores-aria-roleapplication/1252f6ff-ac41-4dd6-9b71-418b6eca40de), FWIW.

**Comment by martynasma** [comment]
Good idea! Thanks.
